### ISSUE TITLE
NaN for loss function, stagnant accuracy

### ISSUE THREAD

**Issue by priyanshuwustl** [gitmenonsandu/Image-Denoising-with-Deep-CNN#2]
I am running the code with the same dataset. However, for some reason the loss function is coming out to be NaN and the accuracy on train and validation sets is also constant. This is after running 10 epochs. I am assuming that if loss function is not defined then no minimization can take place and that is why it is constant.

The only change that I made is that in add_noise, I typecasted batch to int because it was throwing an error that indexed should be integer values.

On displaying the noisy images, it is just showing white_space mostly. Is that correct or is that also maybe a part of the problem?
